app.py

```diff
@@ -40,53 +45,160 @@ def calculate_monte_carlo_var(returns, confidence_level, num_simulations=10000):
         return np.max(returns)
     if confidence_level == 0.0:
         return np.min(returns)
+
     mean = returns.mean()
     std_dev = returns.std()
     simulated_returns = np.random.normal(mean, std_dev, num_simulations)
     var = np.percentile(simulated_returns, (1 - confidence_level) * 100)
     return var
 
+def calculate_cvar(returns, var_threshold):
```

Comment:
Minor: in the future would be good to have some comments on what this function does